docs/dasharo-tools-suite/documentation.md

```diff
@@ -544,6 +544,37 @@ New value is 0x8b.
 SPI Configuration is locked down
 ```
 
+#### Update issues
+
+If you see the following pop-ups during first the boot after the update:
+
+![error-0x03](./images/error-0x03.png)
+![error-0x13](./images/error-0x13.png)
+
+You probably performed an update using a deprecated version of Dasharo Tools
+Suite and have not disabled BIOS lock. Do not worry, nothing bad has happened.
+If you backed up your old firmware, do the following steps:
+
+1. Reboot your device and turn off BIOS lock (you can find this option in
+[Dasharo Security
+Options](../dasharo-menu-docs/dasharo-system-features.md#dasharo-security-options)
+as `BIOS boot medium lock`).
+1. Boot the DTS you backed up your old firmware with and choose option 3, which
+will restore it.
+1. After restoring the firmware, perform another update, but make sure you are
+using the **latest Dasharo Tools suite** version (new DTS has tools to protect
+you from such issues).
+
+In case you did not backup your old firmware, you can download and flash some
+other dasharo firmware for your platform manually as described in [Local
+firmware update](#local-firmware-update) and then perform another firmware
+update, but make sure that you are using the **latest Dasharo Tools suite**
+version (new DTS has tools to protect you from such issues).
```

Comment:
Doesn't that mean one would lose serial numbers, and it would require special recovery to bring that back?

Reply:
@DaniilKl probably the one from SMBIOS https://github.com/Dasharo/meta-dts/blob/main/meta-dts-distro/recipes-dts/dts/dasharo-deploy/dasharo-deploy#L219

Reply:
@pietrushnic, I am not sure that serial numbers are stored under BIOS region of ROM which is being flashed using `flashrom` with `--ifd -i bios`.

Reply:
@DaniilKl Serial numbers are stored in the BIOS region

Reply:
I have edited this section to make it more generic 6f4ac86d81f1b2c6a77368121b8e3d34b7496e82.

Reply:
> Doesn't that mean one would lose serial numbers, and it would require special recovery to bring that back?

IIUC if user flash backed up fw and then perform update (as @DaniilKl suggested) the serial numbers should persist.

@pietrushnic WDYT? I would close this conversation and merge, but you requested changes here so I am unable to do such thing (because of review settings in this repo)

Reply:
@TomaszAIR Sorry, but I don't understand. My question is simple:

> Doesn't that mean one would lose serial numbers, and it would require special recovery to bring that back?

If the answer is yes, then the solution needs a solid explanation as to why we cannot do it the user-friendly way, if the answer is no then I'm ok with resolving the conversation.

Reply:
@macpijan feel free to make final decision

Reply:
> IIUC if user flash backed up fw and then perform update (as @DaniilKl suggested) the serial numbers should persist.

It is true only for automatic update. If user doing it manually via `flashrom` - the numbers should be backed up manually too, which is not described in this documentation.